src/frontend/components/chat-bottom-bar.tsx

```diff
@@ -111,9 +111,14 @@ const ChatBottomBar = ({ onSend }: Props) => {
 				<Button
 					size="icon"
 					variant="default"
-					className="rounded-full flex-shrink-0"
+					className={`rounded-full flex-shrink-0 transition-opacity duration-200 ${
+						message.content.trim()
+							? 'opacity-100'
+							: 'opacity-50'
+					}`}
```

Comment:
Can you make this a state variable? Later on we can extend this to include pending/error states.